tests/model_forms/tests.py

```diff
@@ -1977,7 +1980,8 @@ def test_file_field_data(self):
         # save the file again, but leave it exactly as it is.
         f = TextFileForm(
```

Comment:
single line for some of these? Fine to pass `data` as the first positional arg rather than as a kwarg, I think.